### PR TITLE
DMA

### PR DESCRIPTION
**Pull request by suehtamacv**
Summarize the changes concisely

### Fixed

(Reference to issues, labels, and related merge requests)

## Checklist

- [ ] Automated tests pass
- [ ] Changelog updated
- [ ] Code style guideline is observed

Please check our [contributing guidelines](CONTRIBUTING.md) before opening a Pull Request.

### CODE REVIEW

File: hardware/Makefile [hunk 1]
```diff
@@ -283,7 +284,7 @@ spyglass/tmp/files: $(bender)
 ################
 # Tracing      #
 ################
-benchmark: log simc
+benchmark: log simcvcs
```
Comment (suehtamacv) [inline hardware/Makefile:287]: Any reason why this changed?

File: hardware/scripts/benchmark_dma.fish [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env fish
+
+# Copyright 2021 ETH Zurich and University of Bologna.
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+
+for axi_data_width in 512 256 128
```
Comment (suehtamacv) [inline hardware/scripts/benchmark_dma.fish:7]: Why :fish:?

**Review by suehtamacv — COMMENTED** [review]
Looks okay!

File: hardware/Makefile (hunk 1 above)
Comment (SamuelRiedel) [reply hardware/Makefile:287]: Because vcs is faster

File: hardware/scripts/benchmark_dma.fish (hunk 2 above)
Comment (SamuelRiedel) [reply hardware/scripts/benchmark_dma.fish:7]: Good catch. I replaced it with :snake:.

**Pull request closed without merging**